src/queue/jobs/CallOnValueJob.php

```diff
@@ -89,9 +59,17 @@ public function runStep(int $step)
 			Craft::error("Error during CallOnValue task: " . $e->getMessage());
```

Comment:
"job"

Reply:
Oops thanks for catching that; will fix.